llvm/test/DebugInfo/CAS/AArch64/debug_line_and_info_dedupe.test

```diff
@@ -5,6 +5,8 @@ RUN: llc -O0 -cas-friendly-debug-info --filetype=obj --cas-backend --cas=%t/cas
 RUN: llc -O0 -cas-friendly-debug-info --filetype=obj --cas-backend --cas=%t/cas --mccas-casid %t/b.ll -o %t/b.id
 RUN: llvm-cas-dump --die-refs --cas=%t/cas --casid-file %t/a.id %t/b.id | FileCheck %s
 
+; REQUIRES: arm64
```

Comment:
One other thing to mention is that where target specific tests are in a subdirectory like ARM/ or AArch64/ you can put the requirement in lit.local.cfg to apply it to all tests in that directory (see CodeGen/${TARGET}/lit.local.cfg for an example)